Frends.FixedWidthFlatFile.Parse/Frends.FixedWidthFlatFile.Parse/Definitions/Input.cs

```diff
@@ -19,19 +20,22 @@ public class Input
     /// FixedWidth: Header row is parsed using column specification
     /// Delimited: Header row is parsed using delimiter char
     /// </summary>
+    /// <example>FixedWidth</example>
     [DefaultValue(HeaderRowType.FixedWidth)]
     public HeaderRowType HeaderRow { get; set; }
 
     /// <summary>
     /// If header row uses delimiter set it here.
     /// </summary>
+    /// <example>;</example>
     [UIHint(nameof(HeaderRow), "", HeaderRowType.Delimited)]
     [DisplayFormat(DataFormatString = "Text")]
     public string HeaderDelimiter { get; set; } = string.Empty;
 
     /// <summary>
     /// Column specifications for the data that is being parsed.
     /// </summary>
+    /// <example>Example, String, yyyy-MM-ddTHH:mm:ss, 5</example>
```

Comment:
Example should be in brackets and it should contain the enum should be specified [ Example, ColumnType.String, yyyy-MM-ddTHH:mm:ss, 5 ] to specify that the input needs array.